backend/src/middleware/softDeleteMiddleware.spec.js

```diff
@@ -85,137 +130,182 @@ afterAll(async () => {
 
 describe('softDeleteMiddleware', () => {
   describe('read disabled content', () => {
-    let user
-    let post
-    let comment
+    let subject
     const beforeComment = async () => {
-      query = '{ User(id: "u1") { following { comments { content contentExcerpt }  } } }'
-      const response = await action()
-      comment = response.User[0].following[0].comments[0]
+      graphqlQuery = gql`
+        {
+          User(id: "u1") {
+            following {
+              comments {
+                content
+                contentExcerpt
+              }
+            }
+          }
+        }
+      `
+      const { data } = await action()
+      subject = data.User[0].following[0].comments[0]
     }
     const beforeUser = async () => {
-      query = '{ User(id: "u1") { following { name about avatar } } }'
-      const response = await action()
-      user = response.User[0].following[0]
+      graphqlQuery = gql`
+        {
+          User(id: "u1") {
+            following {
+              name
+              slug
+              about
+              avatar
+            }
+          }
+        }
+      `
+      const { data } = await action()
+      subject = data.User[0].following[0]
     }
     const beforePost = async () => {
-      query =
-        '{ User(id: "u1") { following { contributions { title image content contentExcerpt }  } } }'
-      const response = await action()
-      post = response.User[0].following[0].contributions[0]
-    }
-
-    action = () => {
-      return client.request(query)
+      graphqlQuery = gql`
+        {
+          User(id: "u1") {
+            following {
+              contributions {
+                title
+                slug
+                image
+                content
+                contentExcerpt
+              }
+            }
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-09-03T15:36:57Z" title="Tuesday, September 3rd 2019, 5:36:57 pm +02:00">Sep 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -85,137 +127,186 @@ afterAll(async () => {
 
 describe('softDeleteMiddleware', () => {
   describe('read disabled content', () => {
-    let user
-    let post
-    let comment
+    let subject
     const beforeComment = async () => {
-      query = '{ User(id: "u1") { following { comments { content contentExcerpt }  } } }'
-      const response = await action()
-      comment = response.User[0].following[0].comments[0]
+      graphqlQuery = gql`
+        {
+          User(id: "u1") {
+            following {
+              comments {
+                content
+                contentExcerpt
+              }
+            }
+          }
+        }
+      `
+      const { data } = await action()
+      subject = data.User[0].following[0].comments[0]
     }
     const beforeUser = async () => {
-      query = '{ User(id: "u1") { following { name about avatar } } }'
-      const response = await action()
-      user = response.User[0].following[0]
+      graphqlQuery = gql`
+        {
+          User(id: "u1") {
+            following {
+              name
+              slug
+              about
+              avatar
+            }
+          }
+        }
+      `
+      const { data } = await action()
+      subject = data.User[0].following[0]
     }
     const beforePost = async () => {
-      query =
-        '{ User(id: "u1") { following { contributions { title image content contentExcerpt }  } } }'
-      const response = await action()
-      post = response.User[0].following[0].contributions[0]
+      graphqlQuery = gql`
+        {
+          User(id: "u1") {
+            following {
+              contributions {
+                title
+                slug
+                image
+                content
+                contentExcerpt
+              }
+            }
+          }
+        }
+      `
+      const { data } = await action()
+      subject = data.User[0].following[0].contributions[0]
     }
 
     action = () => {
```


can you define `action` as a `const` outside of the describe block or would this mess everything up? and could you give it a more descriptive name – `sendQuery` or something?